src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,18 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\n Authorization for identifier {authzr.body.identifier} failed.'
+                       msg += '\n Here are the challenges that were not fulfilled:'
+                       for challenge in authzr.body.challenges:
+                         msg += \
+                               f'\n Challenge Type: {challenge.chall.typ}' \
+                               f'\n Error information: ' \
+                               f'\n Type: {challenge.error.typ if challenge.error else ""}' \
+                               f'\n Details: {challenge.error.detail if challenge.error else ""} \n\n'
```

Comment:
@william-gr Good catch, this should remove all spaces after a new line
```suggestion
                       msg += f'\nAuthorization for identifier {authzr.body.identifier} failed.'
                       msg += '\nHere are the challenges that were not fulfilled:'
                       for challenge in authzr.body.challenges:
                         msg += \
                               f'\nChallenge Type: {challenge.chall.typ}' \
                               f'\nError information: ' \
                               f'\nType: {challenge.error.typ if challenge.error else ""}' \
                               f'\nDetails: {challenge.error.detail if challenge.error else ""} \n\n'
```